core/src/main/java/org/infinispan/configuration/cache/HashConfiguration.java

```diff
@@ -184,4 +185,9 @@ public int hashCode() {
       result = prime * result + ((attributes == null) ? 0 : attributes.hashCode());
       return result;
    }
+
+   @Override
+   public boolean matches(HashConfiguration other) {
```

Comment:
Hrmm is this the only one that should override matches for the `Configuration` classes? How is this used btw?

Reply:
No, in the future I plan to add more: this should be sufficient for now to demonstrate the capability